.gitignore

```
@@ -134,3 +134,6 @@ dist
 
 package-lock.json
 yarn.lock
+
+# VS Code configuration
+.vscode
```

```suggestion
.vscode
*.code-workspace
```

Could you also add `*.code-workspace`? It’s also used for VS Code configurations.

Reference: https://github.com/eslint/markdown/pull/362